blocks/workspace.js

```diff
@@ -395,10 +361,10 @@ class ScriptsWorkspace extends Workspace {
     this._vertScrollbar = new Scrollbar(this, false)
   }
 
-  acceptDrop (...args) {
-    return super.acceptDrop(...args)
-      .then(() => this.updateScroll())
-  }
+  // acceptDrop (...args) {
+  //   return super.acceptDrop(...args)
+  //     .then(() => this.updateScroll())
+  // }
```

Comment:
Is this still needed? Try dropping a block along the edges of the workspace or a reporter into a long stack block such that it extends the scroll area, and see if it updates immediately.